onnxruntime/core/platform/windows/env.cc

```diff
@@ -141,6 +141,30 @@ class WindowsEnv : public Env {
     return processorCoreCount;
   }
 
+  std::vector<size_t> GetThreadAffinityMasks() const override {
+    auto generate_vector_of_n = [](int n) {
+      std::vector<size_t> ret(n);
+      std::iota(ret.begin(), ret.end(), 0);
```

Comment:
Instead of return 0, 1, 2, ..., this function shall generate bit mask like static_cast<DWORD_PTR>(1) << i for Windows.

Need handle special case like n = 0, n > 64 or n > logical cores in current processor group.

Since this is a bug existing before this PR, it can be added to back log to fix later.

Reply:
I think std::thread::hardware_concurrency() should never return 0.

Reply:
We need to think how to handle N > 64.

Reply:
std::thread::hardware_concurrency can return zero. It returns zero when the runtime cannot compute a good number. We had bugs about this before I think.

Reply:
Thank you. We'll fix it.

Reply:
Ok, logged a task here https://msdata.visualstudio.com/Vienna/_workitems/edit/722456